pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -246,6 +247,20 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 				return nil, fmt.Errorf("conversion of timeout %s to int failed", fields[len(fields)-1])
 			}
 			conn.Timeout = uint32(val)
+		case strings.Contains(fs, "labels"):
+			fields := strings.Split(fs, "=")
+			labelStr := strings.Replace(fields[len(fields)-1], "0x", "", -1)
+			if len(labelStr) < 16 {
+				labelStr = strings.Repeat("0", 16-len(labelStr)) + labelStr
+			}
+			hexval, err := hex.DecodeString(labelStr)
+			if err != nil {
+				return nil, fmt.Errorf("conversion of labelStr %s to []byte failed", labelStr)
+			}
+			for i := 0; i < len(hexval)/2; i++ {
+				hexval[i], hexval[len(hexval)-i-1] = hexval[len(hexval)-i-1], hexval[i]
+			}
```

Comment:
Sure, the main reason is because the kernel side's label output is little endian, and we would like to keep the logics in connections.go unchanged: https://github.com/antrea-io/antrea/blob/335e6bb770c37847348653871785abc0325e4530/pkg/agent/flowexporter/connections/connections.go#L175

Reply:
That's a good comment to add. BTW I am not a big fan that we assume that we get little-endian labels from the kernel / netlink. CTA_LABELS just uses the "native" endianness for the system, which happens to be little-endian for us. If we were to test Antrea on a big endian CPU, this assumption would break and we would get wrong labels. Not that we test or claim support for such CPUs (although I think ARM CPUs can be used in big-endian mode).

Reply:
Got it, I could open an issue to track this unappropriate assumption.

Reply:
sounds good to me. You may want to check with @srikartati as well, I didn't research this in depth

Reply:
That is a good point, @antoninbas.
Most of the modern CPUs based on x86 architecture are little-endian like Intel, AMD, etc. As Antonin suggested, there are definitely big-endian CPUs like ARM, PowerPC, etc. I see that ARM is bi-endian with default as little-endian through google search.
In any case, we should detect the endianness of the system to cover all scenarios rather than having implementation for all architectures--this should be simple. We can do this when initializing conntrack dumper. IMO this does not warrant an issue. We can have a comment in the conntrack_linux.go code or fix it in this PR.

Reply:
Thanks Srikar, I added a comment in conntrack_linux.go.